spotlight/src/main/java/com/takusemba/spotlight/Spotlight.java

```diff
@@ -126,6 +128,15 @@ public Spotlight setOnSpotlightStartedListener(
     }
 
     /**
+     * listener for target turned up or turned down.
+     */
+    public Spotlight setOnTargetStateChangeListener(
+            @NonNull final OnTargetStateChangeListener listener) {
```

Comment:
if its a listener for a target, should it be set in each target?
like not in a Spotlight itself.

```
someTarget.setOnStateChangeListener(){
    @Override
     public void onStarted() {

     }
    @Override
     public void onEnded() {

     }
}

```